utils/utils.jsx

```diff
@@ -733,9 +732,9 @@ export function applyTheme(theme) {
         changeCss('.app__body .post .post-collapse__show-more-line', `background-color:${changeOpacity(theme.centerChannelColor, 0.1)}`);
 
         if (theme.centerChannelBg) {
-            const ownPostBg = blendColors(theme.centerChannelBg, theme.centerChannelColor, 0.05);
-            const hoveredPostBg = blendColors(theme.centerChannelBg, theme.centerChannelColor, 0.08);
-            const hoveredPostBgLight = blendColors(theme.centerChannelBg, theme.centerChannelColor, 0.05);
+            const ownPostBg = theme.centerChannelBg;
```

Comment:
Instead of just changing the colour here, could you remove the logic that refers to `ownPostBg`? This part of the code is really complicated, so removing that case would simplify it a fair bit.